### ISSUE TITLE
Many Write-offs are in In Process (BC365 and Snowfig Syncing Issues)

### ISSUE THREAD

**Issue by Prakash3K** [3ktech/General#1180]

**Comment by TabrejAlam3K** [comment]
Hi @rohit3k - Is it Deployed into the Production?

CC : @audhi3k @Shasthara @sandhya3k

**Comment by TabrejAlam3K** [comment]
Deployment has been completed into the Production, We can close the ticket from the Support Portal. I am closing from the Git.

CC : @audhi3k @Shasthara @Prakash3K

**Comment by TabrejAlam3K** [comment]
Hi Mr. @rohit3k - Please update why it is not yet deployed into the Production. Any dependencies ?

CC : @audhi3k @Shasthara

**Comment by TabrejAlam3K** [comment]
Hi Mr. @Prakash3K - We can close this ticket form the Git, You can connect with Mr. @Shasthara and close the ticket from the Support portal.

CC : @audhi3k

**Comment by TabrejAlam3K** [comment]
We have already shared our comment to the Client on each points.
1.	Tabrej would check with respect to old RTB, CRTB and write-off records which are still appearing in Snowfig as in Process but has already been posted in BC365, whether the respective records are getting pushed to SB queue or not. 
Comments – We are pushing all the records to SB from the 16th of Feb 2024. For rest of records, we need to push it manually.

**22 Feb 2024** – Mr. Dipesh observed from 16th of Feb there are no entries pending for Snowfig updates, before 16th Feb we found few records, we need to push it manually.

2.	It was also identified that there is a write-off request with RTB number 0000152357 which has been pushed to BC365 but accounts team is not able to access the write-off record in BC365. As checked by Tabrej in logs that the request has been received in BC.
Comments – We are not receiving the Document No. against the WriteOff Request. 

**22 Feb 2024** – Mr. Dipesh Confirmed Snowfig Changes was not deployed on Production, that was the reason WriteOff request reached at BC, but records not get created in “Gen. Journal Lines”.

3.	It was identified that there is a payment received from client against the RTB number [0000881403](https://lcms.snowfig.com/lcmsadmincasefee/newrtb/204665/change/?_changelist_filters=q%3D5048430), but Snowfig system is yet not updated, since the RTB was created in October, so after status API scheduled run the correct status must have been updated, but it is still the same.
Comments – Doc URL is missing on Azure Portal

**22 Feb 2024** – Doc is not yet uploaded over the Azure Portal, It is the reason status does not go back to LCMS.

CC : @audhi3k @Shasthara
